bundles/persistence/org.openhab.persistence.jpa/java/org/openhab/persistence/jpa/internal/JpaPersistenceService.java

```diff
@@ -106,7 +108,9 @@ public void store(Item item, String alias) {
 		
 		JpaPersistentItem pItem = new JpaPersistentItem();
 		try {
-			pItem.setValue(StateHelper.toString(item.getState()));
+			String newValue = StateHelper.toString(item.getState());
+			pItem.setValue(newValue);
+			logger.debug("Storing new value: " + newValue);
```

Comment:
Performance. With string concatenation, you produce lots of objects which get thrown away if the debug is not needed  (by configuration in logback.xml )

If you use the parametrized call, all those strings are only created if necessary . Garbage objects can really grow if you have a long running daemon process like openhab.